examples/low-level-api/create_vc.rs

```diff
@@ -37,17 +41,33 @@ pub async fn create_vc() -> Result<()> {
 
   println!("Credential JSON > {:#}", credential);
 
-  // Validate the verifiable credential
+  // Before passing this credential on the issuer wants to validate that some properties
+  // of the credential satisfy their expectations.
+  // In order to validate a credential the issuer's DID Document needs to be resolved.
+  // Since the issuer wants to issue and verify several credentials without publishing updates to their DID Document
+  // the issuer decides to resolve their DID Document up front now so they can re-use it.
+
   let resolver: Resolver = Resolver::new().await?;
-  let validation: CredentialValidation = common::check_credential(&resolver, &credential).await?;
-  println!("Credential Validation > {:#?}", validation);
-  assert!(validation.verified);
+  let resolved_issuer_doc: ResolvedIotaDocument = resolver.resolve(issuer_doc.id()).await?;
```

Comment:
We don't resolve the issuer document in the Wasm example and I don't think it's necessary here either, honestly. I get that this is more technically correct but it's only necessary in the case when an issuer DID has multiple controllers or if they publish updates incorrectly. I don't really want developers to think they have to resolve their own document for every credential they publish.

Reply:
I forgot to change it here as well.